pkg/cmd/reset/reset.go

```diff
@@ -0,0 +1,52 @@
+package reset
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/MakeNowJust/heredoc"
+	msg "github.com/aziontech/azion-cli/messages/reset"
+	api "github.com/aziontech/azion-cli/pkg/api/personal_token"
+	"github.com/aziontech/azion-cli/pkg/cmdutil"
+	"github.com/aziontech/azion-cli/pkg/logger"
+	"github.com/aziontech/azion-cli/pkg/token"
+	"github.com/spf13/cobra"
+)
+
+func NewCmd(f *cmdutil.Factory) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   msg.USAGE,
+		Short: msg.SHORTDESCRIPTION,
+		Long:  msg.LONGDESCRIPTION,
+		Example: heredoc.Doc(`
+		$ azion logout --help
+        `),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			settings, err := token.ReadSettings()
+			if err != nil {
+				return err
+			}
+
+			if settings.UUID != "" {
+				client := api.NewClient(f.HttpClient, f.Config.GetString("api_url"), f.Config.GetString("token"))
+				err = client.Delete(context.Background(), settings.UUID)
+				if err != nil {
+					return fmt.Errorf(msg.ERRORLOGOUT, err.Error())
+				}
+			}
+
+			settings = token.Settings{}
+			err = token.WriteSettings(settings)
+			if err != nil {
+				return err
+			}
+
+			logger.LogSuccessBye(f.IOStreams.Out, msg.SUCCESS)
```

Comment:
using outupt.Print would make the output manageable

Reply:
```	
creatOut := output.GeneralOutput{
		Msg:   msg.OUTPUT_CREATE_BUCKET,
		Out:   f.IOStreams.Out,
		Flags: f.Flags,
}
return output.Print(&creatOut)
```

Reply:
True that. Fixed for both reset and logout